index.html

```diff
@@ -26,24 +24,259 @@ <h1>Hey there. I'm Richard</h1>
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+
+   </div>
+
+   <div class="about-me">
+     <section class="aboutme-title">
+        <h3>About Me</h3>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+don’t hestiate to contact me.</p>
+        <button class="about-btn" type="button" name="button">Get My Resume</button>
+     </section>
+
+     <ul>
+        <li class="top-skill">
+            <img src="images/icon-languages.png" alt="">
+
+            <div class="skill-title">
+                <p>Languages</p>
+            </div>
+
+            <div class="container-skill">
+                <div class="container-skill-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-skill-element">
+                   <a href="#">Ruby</a>
+                </div>
+                <div class="container-skill-element">
+                   <a href="#">Html</a>
+                </div>
+                <div class="container-skill-element">
+                   <a href="#">CSS</a>
+                </div>
+            </div>
+
+        </li>
+
+        <li class="top-skill">
+            <img src="images/icon-frameworks.png" alt="">
+
+            <div class="skill-title">
+                <p>Frameworks</p>
+            </div>
+
+            <div class="container-skill">
+                <div class="container-skill-element">
+                   <a href="#">Bootstrap</a>
+                </div>
+                <div class="container-skill-element-mid">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-skill-element">
+                   <a href="#">Rspec</a>
+                </div>
+            </div>
+            <div class="container-skill-2">
+              <div class="container-skill-element">
+                 <a href="#">Capybara</a>
+              </div>
+              <div class="container-skill-element">
+                 <a href="#">Selenium</a>
+              </div>
+            </div>
+
+        </li>
```

Comment:
- [ ] Frameworks are centered in the original design and margin is less between each other. For stick to design please fix it. Also, check the skills and languages section too.

YOUR DESIGN
![image](https://user-images.githubusercontent.com/38635158/148471166-ba3ef326-92c8-492d-bb03-606fce2620be.png)

ORIGINAL DESIGN
![image](https://user-images.githubusercontent.com/38635158/148471152-bad7da80-430e-4f20-89e7-0a19f2a0faa4.png)